stripe-ui-core/src/main/java/com/stripe/android/uicore/address/AddressRepository.kt

```diff
@@ -39,7 +39,7 @@ class AddressRepository @Inject constructor(
         schema: ByteArrayInputStream
     ) = initialize(mapOf(countryCode to parseAddressesSchema(schema)!!))
 
-    private fun initialize(
+    fun initialize(
```

Comment:
Why are we exposing? Does this require an API review?

Reply:
no need, reverted